theseus/utils/examples/bundle_adjustment/data.py

```diff
@@ -0,0 +1,340 @@
+from typing import List, Optional, cast
+
+import numpy as np
+import torch
+
+import theseus as th
+
+from .util import random_small_quaternion
+
+
+class Camera:
```

Comment:
Move Camera and ReprojectionError to embodied?

Maybe not urgent as we prioritize experiments but also not sure how much we should punt as 'cleanup' for later.

Reply:
My hesitation is that I would consider the standard for moving to `embodied` higher than keeping it here, and this would mean spending extra time writing a full test suite, possibly refactor the code, then iterating during code review, etc. I would prefer to defer until we have more progress on the next milestone.